trymito.io/pages/use-cases/python-training.tsx

```diff
@@ -0,0 +1,320 @@
+                                    df[&apos;Age&apos;] = df[&apos;Age&apos;].astype(int)
+                                </li>
+                                <li style={{marginTop: '.5rem'}}>
+                                    df[&apos;Age&apos;] = df[&apos;Age&apos;].as_type(int)
+                                </li>
+                            </ol>
+                            <p className={pythonTrainingStyles.quiz_answer_button} onClick={() => setShowQuizAnswer(true)}>
+                                Reveal the right answer
+                            </p>
+                        </div>
+                    </div>
+                </section>
+
+                <section className='center'>
+                    <div className={pythonTrainingStyles.training_quote_container}>
+                        <p className='quote'><span className={classNames(customerCardStyles.quote_symbol, pythonTrainingStyles.training_quote_text)}>❝We&apos;ve trained thousands of analysts to build Python automations with Mito. Every time we make Mito a bigger part of our training, our end user reviews go up. Now Mito is the first thing we teach in our Python trainings. </span></p>
+                        <p>
+                            Chief Data Architect @ Bulge Bracket Bank
+                        </p>
+                    </div>
+                </section>
+
+                <section>
+                    <div className={classNames(pageStyles.subsection, pageStyles.subsection_justify_baseline)}>
+                        <div className={textImageSplitStyles.functionality_text}>
+                            <h2>
+                                Track Python Adoption. Open the Black Box.
+                            </h2>
+                            <p> 
+                                Mito lets you track your organization&apos;s Python adoption so you can easily evaluate the success of your trainings and communicate it&apos;s value with your organization&apos;s leadership.
+                            </p>
+                            <p>
+                                Mito has built in logging infrastructure that lets you track metrics like: how many analysts are writing and re-running Python code, which database integrations are most popular, and which automations are shared across the firm. 
+                            </p>
+                            <p>
+                                Just point Mito to your enterprise logging infrastructure and we&apos;ll take care of the rest. No data will ever leave your computers, of course.
+                            </p>
+                        </div>
+                        <div className={classNames(textImageSplitStyles.functionality_media, textImageSplitStyles.functionality_media_supress_bottom_margin)}>
+                            <Image src={'/python-training/black-metrics-box.png'} alt='Mito Python Adoption Logging' width={457} height={457} layout='responsive'/>
```

Comment:
This image does not seem to communicate what we want. I don't wanna die of exploding metrics. I want to understanding the few important metrics I have around Python adoption. 

Maybe a funnel diagram would be better suited here